cmd/cloud/root.go

```diff
@@ -23,10 +24,14 @@ func AddCmds(astroPlatformCoreClient astroplatformcore.CoreClient, coreClient as
 	platformCoreClient = astroPlatformCoreClient
 	astroCoreIamClient = iamCoreClient
 	airflowAPIClient = airflowClient
-	return []*cobra.Command{
+	cmds := []*cobra.Command{
 		NewDeployCmd(),
 		newDeploymentRootCmd(out),
 		newWorkspaceCmd(out),
 		newOrganizationCmd(out),
 	}
+	if config.CFG.DbtDeploysEnabled.GetBool() {
```

Comment:
I dont think we need this, since this is behind a feature flag anyway, and folks trying to use this without the feature flag would be thrown the appropriate error.

Reply:
I think it's a good idea to keep the command hidden by default until the feature is at least in public preview. We did it like this for environment objects and it worked well.

Reply:
We shouldnt be hiding the command...If folks run it they would see the error message and if they are curious and want to opt in, thats a good sign for us. Plus for anyone who wants to opt in, i dont see a knob for 2 configurations to be enabled for them to use this...

Reply:
I've removed the config and the command is now always added.